src/style.css

```diff
@@ -1 +1,75 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+}
+
+body {
+  padding: 0;
+}
+
+.search-bar {
+  margin: 20px 0;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
```

Comment:
It's good practice to include a fallback font family in case the main one doesn't load. For example, `font-family: Avenir, Arial, sans-serif;`.